kadai3-1/typegame/typegame.go

```diff
@@ -0,0 +1,150 @@
+package typegame
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"io"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"strconv"
+	"time"
+)
+
+// Game タイピングゲームを管理する構造体
+type Game struct {
+	IReader
+	TimeLimit   int
```

Comment:
time.Durationにしておくといろいろ楽です
```suggestion
	TimeLimit   time.Duration
```

Reply:
修正しました！
ありがとうございます！